src/functional/ffront/dialect_parser.py

```diff
@@ -113,6 +113,12 @@ def apply_to_function(
         closure_refs = ClosureRefs.from_function(func)
         return cls.apply(source_definition, closure_refs, externals)
 
+    def generic_visit(self, node: ast.AST) -> None:
+        raise self._make_syntax_error(
+            node,
+            message=f"Nodes of type {type(node).__module__}.{type(node).__name__} not supported in dialect.",
```

Comment:
See: https://docs.python.org/3.10/glossary.html#term-qualified-name

```suggestion
            message=f"Nodes of type {type(node).__module__}.{type(node).__qualname__} not supported in dialect.",
```